### ISSUE TITLE
adding bodyparts to label

### ISSUE THREAD

**Issue by fidelDLC** [DeepLabCut/DeepLabCut#2565]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

In the _label frames_ part the napari program allowed me to use only 4 points layer, no more. I read the napari tutorial but I found nothig that help me to add more ponts into my video frame, also the _new points layer_ button is blocked.

### Operating System

Windows 10

### DeepLabCut version

2.3.9

### DeepLabCut mode

single animal

### Device type

Geforce RTX 4060

### Steps To Reproduce

1.Using Conda Environment
2. Load in Deeplabcut gui
3. Load Project - from .yaml in project folder
4. Go to label frames tab
5. select any folder that contains extracted frames
6. the napari window opens in the first frame extracted
7. click on add points
8. just allowed me to add 4 points

### Relevant log output

```shell
No error appears in the anaconda powershell
```

### Anything else?

![image](https://github.com/DeepLabCut/DeepLabCut/assets/167367948/7b5f8a15-3c75-446d-9734-d95511942a1a)

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by n-poulsen** [comment]
Hi @fidelDLC, **before** starting to label data you need to edit the project's "config.yaml" file and indicate all bodyparts you want to label. If you open the `config.yaml` file (in the project folder), you should see something like:

```yaml
    # Project definitions (do not edit)
Task: myproject
...

bodyparts:
- bodypart1
- bodypart2
- bodypart3
```

To label more bodyparts, simply add the bodyparts you want to label:

```yaml
bodyparts:
- eyeLeft
- eyeRight
- nose
- tailBase
- tailEnd
```

For more information on labeling data, you can also check out our [single animal user guide](https://deeplabcut.github.io/DeepLabCut/docs/standardDeepLabCut_UserGuide.html#b-configure-the-project)!

**Comment by fidelDLC** [comment]
Hi @n-poulsen, Thank you for your answer. I made what you told me but the program does not respond. I added the body parts I needed using the .yaml editor in the DLC window and it went like this: 
![image](https://github.com/DeepLabCut/DeepLabCut/assets/167367948/0f4b4115-4d6f-40b8-b639-6133c219cb84)

![image](https://github.com/DeepLabCut/DeepLabCut/assets/167367948/32b45834-6fef-4631-aa84-e153cd684c1b)

also I used the Github editor (ATOM) but the same. the program stop working. I don't know what I am doing wrong.

Thank you!
